web/src/main/java/com/garret/movies/controller/MovieDatabaseController.java

```diff
@@ -0,0 +1,61 @@
+package com.garret.movies.controller;
+
+import com.garret.movies.dao.entity.Movie;
+import com.garret.movies.service.api.MovieService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/db")
```

Comment:
@Garret-vin Please read about Rest API best practices and naming conversions https://restfulapi.net/resource-naming/